### PR TITLE
Feature seed period

### PR DESCRIPTION
**Pull request by philrwebdev**
This is the PR for the ['seed period' user story ](https://github.com/users/SabinePfanner/projects/2/views/1?pane=issue&itemId=65503925). It allows to define a seed period for a given plant when creating a new plant or editing an existing plant (bottom of form) via a custom timeline. 

Please note that I had a local issue with git last week, with a history restore that somehow went wrong, so this feature branch was continued as an old feature branch. Please ignore the first few commits shown below, these are not part of the current PR.

### CODE REVIEW

File: components/CardList.js [hunk 1]
```diff
@@ -104,6 +104,26 @@ export default function CardList({
...
+  //     );
+  //     idx++;
+  //     return filterPlants(filteredPlants, filters);
+  //   } else {
+  //     return plants;
+  //   }
+  // }
+
+  // const filteredPlants = filterPlants(plants, filter);
+
```
Comment (janaRicarda) [inline components/CardList.js]: please delete commented code

File: components/Form.js [hunk 2]
```diff
@@ -66,6 +72,10 @@ export default function Form({
     frostSensitive: true,
     waterDemand: 2,
     nutrientDemand: 2,
+    tasks: {
+      // seed: { start: "Early March", end: "Late March" },
```
Comment (janaRicarda) [inline components/Form.js]: same as above

File: components/Form.js [hunk 3]
```diff
@@ -259,6 +285,26 @@ export default function Form({
...
+            : `${seedPeriod.seed.start} \u2013 ${seedPeriod.seed.end}`}
+        </StyledPeriodSubheader>
+      </Label>{" "}
+      <TaskPeriod
+        task={data.tasks}
+        taskName="seed"
+        onSeedPeriod={handleSeedPeriod}
+        // tasks={{
+        //   seed: { start: null, end: null },
+        // }}
```
Comment (janaRicarda) [inline components/Form.js]: commented code...

File: components/StyledElements/HoverTooltip.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  bottom: 70%;
+  left: 50%;
+  margin-left: -60px;
+
+  /* Fade in tooltip */
+  opacity: 0;
+  transition: opacity 0.3s;
+
+  &:hover {
+    /* visibility: visible; */
```
Comment (janaRicarda) [inline components/StyledElements/HoverTooltip.js]: to be deleted

File: components/StyledElements/MultiSelect.js [hunk 5]
```diff
@@ -114,7 +114,6 @@ export default function MultiSelectDropdown({
                     toggleOption(category, option);
                   }
                   if (event.key === "Tab") {
-                    console.log(event);
```
Comment (janaRicarda) [inline components/StyledElements/MultiSelect.js:117]: please remove console.logs

File: components/TaskPeriod.js [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+  background-color: ${(props) =>
+    props.$alternateBackground ? "#657383" : "#97A5B5"};
+  color: white;
+  font-size: 1rem;
+  border-right: 0.05rem solid white;
+`;
+
+const StyledInterval = styled.div`
+  display: flex;
+  /* position: relative; */
```
Comment (janaRicarda) [inline components/TaskPeriod.js]: delete, if not needed

File: components/TaskPeriod.js [hunk 7]
```diff
@@ -0,0 +1,324 @@
...
+
+const StyledSvgIcon = styled(SvgIcon)`
+  min-height: 25px;
+`;
+
+const StyledDummySection = styled.section`
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  /* align-items: center; */
```
Comment (janaRicarda) [inline components/TaskPeriod.js]: delete if not needed

File: styles.js [hunk 8]
```diff
@@ -7,6 +7,31 @@ export default createGlobalStyle`
...
+  scrollbar-color: darkgrey lightgrey;
+}
+
+/* Works on Chrome, Edge, and Safari */
+*::-webkit-scrollbar {
+  width: 1rem;
+}
+
+*::-webkit-scrollbar-track {
+  /* background: orange; */
```
Comment (janaRicarda) [inline styles.js]: remove if not needed

**Review by janaRicarda — CHANGES_REQUESTED** [review]

**Comment by janaRicarda** [comment]
Hey, very nice feature! I left you some comments, because there is a lot of commented code, that should be deleted. But everything else is fine!

**Comment by philrwebdev** [comment]
Hi Jana, many thanks for your review and for spotting all the remnant pieces of unused code, I'll clean this up!

**Pull request merged**